myapp/templates/password_change.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+
+{% block content %}
+<div class="container">
+    <div class="col-md-12 text-left">
+        <h3>Change password for <b>{{ user }}</b></h3><br>
+    </div>
+    <div class="form-group input-group">
+        <form method="POST">
+            {{ form.non_field_errors }}
+            {% csrf_token %}
+            {% for field in form %}
```

Comment:
just a small note (maybe you already familiar with), is using  `{{form.as_p}}` giving somewhat the same form

Reply:
Yeah, I wrote it like this instead of `{{form.as_p}}` to be able to individually access the fields/help text within the form and customize it a bit using CSS